ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -27,38 +27,36 @@
       // Particle and spell effects in the room can make this Dark Mist hard to see.
       // Callout ensures dodge.
       id: 'Haukke Normal Amandine Dark Mist Dodge',
-      netRegex: NetRegexes.startsUsing({ id: '2C1', source: 'Lady Amandine', capture: false }),
-      condition: function(data) {
-        return !data.canStun();
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: 'Lady Amandine' }),
+      condition: function(data, matches) {
+        return !data.CanStun();
       },
       response: Responses.outOfMelee('alert'),
```

Comment:
Here, unlike your other  triggers, there shouldn't be a need to capture. `outOfMelee()` takes no arguments, so `matches` won't be needed there, and as noted elsewhere, you don't need `matches` to check whether a job can stun/silence.

(As a general rule, we try to avoid superfluous function parameters where possible. Basically, if a parameter is not used, and it is not "in front of" a parameter that is being used, you can safely not include it.)